script/upgrades/MU01/deploy/MU01-04-Deploy-BiPoolMananager.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.5.13;
+
+import { Script } from "script/utils/Script.sol";
+import { Chain } from "script/utils/Chain.sol";
+import { console2 } from "forge-std/Script.sol";
+
+import { BiPoolManager } from "mento-core/contracts/BiPoolManager.sol";
+
+/*
+ forge script MU01_DeployBiPoolManager --rpc-url $RPC_URL
+                             --broadcast --legacy 
+                             --verify --verifier sourcify 
+*/
+contract MU01_DeployBiPoolManager is Script {
+  function run() public {
+    address biPoolManager;
+
+    vm.startBroadcast(Chain.deployerPrivateKey());
+    {
+      // Updated implementation
+      biPoolManager = address(new BiPoolManager(false));
```

Comment:
Let's also transfer the implementation's ownership to Governence here.

Reply:
added a line for that. if it looks good I'll rerun the script.